javascript/ql/test/query-tests/Security/CWE-843/tst.js

```diff
@@ -100,7 +100,7 @@ express().get('/foo', function (req, res) {
 });
 
 express().get('/foo', function (req, res) {
-    let data = req.query;
+    let data = req.query.data;
```

Comment:
```suggestion
    let data = req.query.data;
    data.indexOf(); // NOT OK
```
Let us include an example we actually flag. 
That's why the bug wasn't found originally.

Reply:
Good point 👍